inst/components/scss/sidebar.scss

```diff
@@ -11,12 +11,13 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
   --bslib-sidebar-bg: #{$bslib-sidebar-bg};
   --bslib-sidebar-fg: #{$bslib-sidebar-fg};
   --bslib-sidebar-padding: calc(var(--bslib-spacer) * 1.5);
+  --bslib-sidebar-content-padding: var(--bslib-sidebar-padding, 1.5rem);
```

Comment:
Assuming we go the direction of https://github.com/rstudio/bslib/pull/725/files#r1284825876

```suggestion
```